src/test/java/wolox/training/mocks/InvalidBook.java

```diff
@@ -0,0 +1,57 @@
+package wolox.training.mocks;
+
+import wolox.training.models.Book;
+
+public class InvalidBook extends Book {
```

Comment:
There is no need for this class, because you're not adding ant logic to the model of Book. The method bookWithoutAuthor can be part of either the test class or a TestUtilities class 🙂